### ISSUE TITLE
In 'validate_model', only 'with torch.no_grad()'

### ISSUE THREAD

**Issue by jianzhuwang** [ugent-korea/pytorch-unet-segmentation#18]
I do not know why  'model.eval()' is not used in 'validate_model'.

In my own opinion, there should be a 'model.eval()' in both validatation and testing stage.

**Comment by PyeongKim** [comment]
Yes, in general you are right.
However, in our case, we did not use any of dropout nor batch normalization in our model.
Thus, it did not bother to use model.eval() which behaves differently from training in case of presence of dropout or batch_norm.
